src/models/character.ts

```diff
@@ -0,0 +1,43 @@
+import { DefaultTags } from '@models/tags'
+import { jsonArrayMember, jsonMember, jsonObject } from 'typedjson'
+import { Trait } from './trait'
+
+@jsonObject({
+  initializer: () => {
+    // eslint-disable-next-line @typescript-eslint/no-use-before-define
+    return new Character(true)
+  },
+})
+export class Character {
+  @jsonMember({ constructor: Number })
```

Comment:
Brauchen wir das `({ constructor: Number })` nicht dank reflect-metadata nicht mehr?

```suggestion
  @jsonMember
```